pkg/utils/progress.go

```diff
@@ -6,20 +6,32 @@ import (
 	"context"
 	"encoding/json"
 	"io"
+	"sync/atomic"
 	"time"
 
 	"github.com/cheggaaa/pb/v3"
 	"github.com/pingcap/log"
 	"go.uber.org/zap"
 )
 
+type atomicAdder int64
+
+func (a *atomicAdder) add(n int64) int64 {
+	return atomic.AddInt64((*int64)(a), n)
+}
+
+func (a *atomicAdder) get() int64 {
+	return atomic.LoadInt64((*int64)(a))
+}
+
 // ProgressPrinter prints a progress bar.
 type ProgressPrinter struct {
 	name        string
 	total       int64
 	redirectLog bool
+	progress    atomicAdder
```

Comment:
i think using `int64` directly here is fine? using a new type here does not provide any real protection as `progress += 1` is still accepted even using the `atomicAdder` type.

Reply:
`progress.add(1)` will be shorter than `atomic.AddInt64(&progress, 1)`, and more semantic...?

If more protection is needed, I can wrap it like `type atomicAdder struct { inner int64 }`.

Reply:
it's shorter in the call, but you need to define an entire type and function which is only used once. i don't think it is more semantic here.

Reply:
well, i have removed it.